main.go

```diff
@@ -39,11 +43,18 @@ type AgentStateResult struct {
 	} `json:"agent-state-update"`
 }
 
+type Event struct {
+	Title string
+	Start int
+	End   int
```

Comment:
We use lowercase for the other JSON endpoints. Maybe we should do that here as well?